Ginger/GingerCore/Drivers/Appium/SeleniumAppiumDriver.cs

```diff
@@ -639,9 +701,9 @@ public override void RunAction(Act act)
                 if (DriverPlatformType == eSeleniumPlatformType.AndroidBrowser || DriverPlatformType == eSeleniumPlatformType.iOSBrowser)
                 {
                     if (DriverPlatformType == eSeleniumPlatformType.AndroidBrowser)
-                        DriverWindow.ShowActionEfect(false); //ios connection is crashing if to much screen shots taken
+                        if (DriverWindow!=null) DriverWindow.ShowActionEfect(false); //ios connection is crashing if to much screen shots taken
                     mSeleniumDriver.RunAction(act);
-                    DriverWindow.ShowActionEfect(true, 1500);
+                    if (DriverWindow != null) DriverWindow.ShowActionEfect(true, 1500);
```

Comment:
Codacy found an issue: [Add curly braces around the nested statement(s) in this 'if' block.](https://app.codacy.com/manual/Ginger-Automation/Ginger/pullRequest?prid=6113827)